src/debuglog.js

```diff
@@ -42,19 +42,19 @@ function debuglog (set) {
 	if (debugEnv === undefined) {
 		debugEnv = process.env.NODE_DEBUG || ''
 	}
-	set = set.toUpperCase()
-	if (!debugs[set]) {
-		if (debugEnv === '*' || new RegExp('\\b' + set + '\\b', 'i').test(debugEnv)) {
+	const upperSet = set.toUpperCase()
```

Comment:
I would rather not change this method too much. It's taken directly from Node (https://github.com/nodejs/node/blob/master/lib/util.js#L145).